assets/pages/contributions-landing/newContributionsLanding.scss

```diff
@@ -0,0 +1,333 @@
+}
+
+.countries .icon__arrow-up {
+  display: none;
+}
+
+.countries[open] .icon__arrow-up {
+  display: initial;
+}
+
+.countries[open] .icon__arrow-down {
+  display: none;
+}
+
+.countries summary:hover,
+.countries summary:focus {
+  box-shadow: 0 0 0 3px gu-colour(news-garnett-highlight);
+  color: inherit;
+}
+
+.countries summary::-webkit-details-marker {
+  display: none;
+}
+
+.countries__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.countries__item > a {
+  color: inherit;
+  display: flex;
+  padding: 0 40px;
+  line-height: 40px;
+  text-decoration: none;
+}
+
+.countries__item > a:hover,
+.countries__item > a:focus {
+  background: gu-colour(news-garnett-highlight);
+  font-weight: bold;
+}
+
+.countries__item .icon {
+  margin-left: -40px;
+}
+
+/*= FOOTER */
+
+footer[role="contentinfo"] {
+  background: #333;
+  color: gu-colour(light-black);
+  padding-bottom: $gu-v-spacing * 2;
+}
+
+footer[role="contentinfo"] a {
+  color: white;
+}
+
+/*= CONTENT */
+.gu-content__main {
+  background: white;
+  padding: 0 10px;
+}
+
+/*= TYPOGRAPHY */
+body {
+  color: #333;
+  font: normal 16px/1.5 $gu-text-egyptian-web;
+}
+
+h1 {
+  border-bottom: 1px solid gu-colour(neutral-4);
+  color: #121212;
+  font: bold 24px/1.25 $gu-egyptian-web;
+  margin: 0 -10px;
+  padding: 0 10px $gu-v-spacing;
+}
+ 
+input,
+select,
+textarea,
+button {
+  font: inherit;
+}
+  
+.blurb {
+  font-size: 14px;
+  line-height: 1.4;
+  margin: ($gu-v-spacing / 4) 0 ($gu-v-spacing / 4 * 3);
+}
+
+footer[role="contentinfo"] {
+  font-family: $gu-text-sans-web;
+}
+
+.license__txt {
+  font-size: inherit;
+}
+
+/*= BREAKPOINTS */
+
+header[role="banner"] {
+  height: 75px;
+}
+
+@media (min-width: 740px) {
+  header[role="banner"],
+  footer[role="contentinfo"] {
+    padding: 0 calc(50% - 350px) ($gu-v-spacing * 2);
+    width: 700px;
+  }
+  
+  .gu-content__main {
+    border-color: gu-colour(neutral-4);
+    border-style: none solid;
+    border-width: 0 1px;
+    width: 340px;
+    margin-left: calc(50% - 361px); /* account for the 11px of negative margin */
+  }
+
+  header[role="banner"],
+  footer[role="contentinfo"] {
+    display: flex;
+  }
+
+  header[role="banner"] {
+    align-items: flex-start;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+  }
+  }
+
+@media (min-width: 1140px) {
+  header[role="banner"],
+  footer[role="contentinfo"] {
+    padding: 0 calc(50% - 550px) ($gu-v-spacing * 2) calc(50% - 390px);
+    width: 940px;
+  }
+
+  .gu-content__main {
+    width: 300px;
+    margin-left: calc(50% - 401px); /* account for the 11px of negative margin */
+  }
+}
+
+@media (min-width: 1300px) {
+  header[role="banner"],
+  footer[role="contentinfo"] {
+    padding: 0 calc(50% - 620px) ($gu-v-spacing * 2);
+    width: 1240px;
+  }
+  
+  .gu-content__main {
+    width: 300px;
+    margin-left: calc(50% - 400px); /* account for the 11px of negative margin */
+  }
+  }
+
+@supports(display: grid) {
```

Comment:
does this imply it will look correct without grid? if so, is the idea to remove the legacy non-grid styles at some future date once we revise our browser support? if not, I'd be worried it might influence the result of the test since IE11 is in our list of supported browsers: https://github.com/guardian/support-frontend/blob/master/package.json#L19-L31

Reply:
Yeah, I've used flexbox by default and grid kicks in for those modern browsers that do support it. I'm hoping though @michaelwmcnamara can do some QA on my work as I may not have thought about every edge case.